your-first-cluster.md

```diff
@@ -512,14 +508,12 @@ kubectl --kubeconfig cluster-one.kubeconfig apply -f yamls/cni/calico.yaml
 
 With the deployment created, watch for the pods to come up.
 ```bash
-kubectl --kubeconfig cluster-one.kubeconfig get pods -A -w
+kubectl --kubeconfig cluster-one.kubeconfig get nodes,pods -A -w
```

Comment:
This doesn't work for me - can't get multiple resources across all namespaces I think.

Reply:
Can we take a closer look? I'm using this since 1-2 years all the time :)

Reply:
Apologies - it wasn't the namespace selector - the watch flag is returning an error:
```
error: you may only specify a single resource type
```

Reply:
Thx! I'll revert